frontend/src/components/shared/WorkflowSamplesTable/SampleTableColumns.tsx

```diff
@@ -411,6 +478,8 @@ export const SAMPLE_FILTER_KEYS: { [key in SampleColumnID]: string } = {
 	[SampleColumnID.KIND]: 'derived_samples__sample_kind__name',
 	[SampleColumnID.NAME]: 'name',
 	[SampleColumnID.INDIVIDUAL]: 'derived_samples__biosample__individual__name',
+	[SampleColumnID.PARENT_CONTAINER]: 'container__location__barcode',
+	[SampleColumnID.PARENT_COORDINATES]: 'container__location__coordinate__name',
```

Comment:
This filter look wrong ... This should be the end of path as sample_next_step. ...container__coordinate__name.